server/gql/queries.js

```diff
@@ -11,6 +11,7 @@ import { storeProductQueries } from '@gql/models/storeProducts';
 import { storeQueries } from '@gql/models/stores';
 import { supplierQueries } from '@gql/models/suppliers';
 import { supplierProductQueries } from '@gql/models/supplierProducts';
+import { userQueries } from '@gql/models/users';
```

Comment:
similar to this, we need to expose mutations as well.

Reply:
only update mutation should be allowed. delete and create shouldn't be allowed. 

please take a look at this implementation here: https://github.com/wednesday-solutions/pawlyclinic-api/blob/develop/server/gql/mutations.js#L95